resources/property.py

```diff
@@ -57,6 +56,27 @@ def post(self, id):
 
         return property.json(), 201
 
+class ArchiveProperties(Resource):
+    parser = reqparse.RequestParser()
+    parser.add_argument('ids', action='append')
+
+    @admin_required
+    def post(self):
+        data = ArchiveProperties.parser.parse_args()
+        propertyList = []
+
+        for id in data['ids']:
+            property = PropertyModel.find_by_id(id)
+            if(not property):
+                return{'message': 'Properties cannot be archived'}, 400
+            property.archived = True
+            propertyList.append(property)
+
+        for p in propertyList:
+            p.save_to_db()
```

Comment:
We should be able to do something like this:
```suggestion
       db.session.bulk_save_objects(propertyList)
```

Reply:
Thank you for the tip!